app/views/proposals/_submission_type_content_form.haml

```diff
@@ -2,8 +2,11 @@
 %p Please select a submission type, then fill in the abstract and extended details.
 
 .form-group
-  = f.label :event_type_id, 'Type'
-  = f.select :event_type_id, event_type_select_options(@conference.program.event_types), { include_blank: false }, { class: 'select-help-toggle form-control' }
+  = f.label :event_type_id, "Type"
+  - if current_user&.is_admin
+    = f.select :event_type_id, event_type_select_options(@conference.program.event_types), { include_blank: false }, { class: 'select-help-toggle form-control' }
+  - else
+    = f.select :event_type_id, event_type_select_options(@conference.program.event_types.available_for_public), { include_blank: false }, { class: 'select-help-toggle form-control' }
```

Comment:
What'd I'd do here is set a variable `visible_event_types` or something like that, and not duplicate the entire select input line.

Reply:
fixed